object_manager.go

```diff
@@ -304,6 +309,23 @@ func (objMgr *ObjectManager) GetFixedAddress(netview string, cidr string, ipAddr
 	return &res[0], nil
 }
 
+func (objMgr *ObjectManager) UpdateFixedAddress(fixedAddr *FixedAddress, macAddress string, vmID string) (*FixedAddress, error) {
```

Comment:
If `fixedAddr` is the result of `GetFixedAddress()`, then it would have all it's fields populated. However, we might want to ONLY pass fields that are being updated (plus the Ref) to `UpdateObject()` below. Example, do something like:

`    updateFixedAddr := NewFixedAddress(FixedAddress{Ref: fixedAddress.Ref})`

and then use `updateFixedAddr` instead of `fixedAddr` in the code that follows.

An alternative would be to change this method (`UpdateFixedAddress`) to accept a `Ref `instead of `FixAddress `as the 1st argument.